package.json

```diff
@@ -29,25 +29,22 @@
   },
   "homepage": "https://github.com/vaibhavchellani/BOPR-contracts#readme",
   "devDependencies": {
-    "@nomiclabs/buidler": "^1.4.6",
-    "@nomiclabs/buidler-ethers": "^2.0.0",
-    "@nomiclabs/buidler-waffle": "^2.1.0",
+    "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@openzeppelin/contracts": "^2.4.0",
     "@typechain/ethers-v5": "^1.0.0",
-    "@types/chai": "^4.2.12",
-    "@types/mocha": "^8.0.3",
+    "@types/chai": "^4.2.14",
+    "@types/mocha": "^8.0.4",
     "bn.js": "^5.1.3",
     "chai": "^4.2.0",
-    "ethereum-waffle": "^3.1.0",
```

Comment:
Removed Waffle because it depends on ganache, which depends on other horrible things that `npm ci` could not even install. We only use its test helpers, which can be easily replaced with functions from ethers.js.